strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/StoragesConfigurationController.java

```diff
@@ -252,6 +258,13 @@ public ResponseEntity addOrUpdateRepository(@ApiParam(value = "The storageId", r
 
             configurationManagementService.saveRepository(storageId, repository);
 
+            if (repository != null && RepositoryTypeEnum.PROXY.getType().equals(repository.getType()))
```

Comment:
`repository != null` unnecessary check

Reply:
`RepositoryTypeEnum.PROXY.getType().equals(repository.getType())` could you please follow the same validation checks as in `org.carlspring.strongbox.services.impl.ConfigurationManagementServiceImpl#setProxyRepositoryConnectionPoolConfigurations`. Maybe we can extract extra method and use it in both cases.